backend/graph/api/shorts.utils.go

```diff
@@ -187,6 +191,18 @@ func (r *Resolver) getShorts(ctx context.Context, cursor *string, limit *int, in
 	if err != nil {
 		return nil, err
 	}
+	if initialUUID != uuid.Nil {
+		isInResult := false
+		for _, s := range shorts {
+			if s.ID == initialUUID {
+				isInResult = true
+				break
+			}
+		}
+		if !isInResult {
+			graphql.AddError(ctx, ErrItemNotFound)
+		}
+	}
```

Comment:
@KillerX i ended up just doing this instead of GetManyStrict. Made more sense to simply add an error if it's not in the result.